LungCancerDetection/INSTALL.md

```diff
@@ -0,0 +1,56 @@
+## Installation
+
+This file explains how to install SQL Server with python and all the needing libraries to run the lung cancer demo.
+
+#### Python with SQL Server Community Technical Preview
+
+The first step is to download [SQL Server 2017 CTP](https://www.microsoft.com/en-us/evalcenter/evaluate-sql-server-2017-ctp/). One of the most important features of the version is that it has built-in Python and R integration. It comes with Anaconda 4.2.0 (64-bit) with Python 3.5. 
+
+To test that python is working correctly, make sure that the library `revoscalepy` can be loaded. To test it, go to the installation folder and look for `python.exe` inside `PYTHON_SERVICES` folder.
+
+	cd "C:\Program Files\Microsoft SQL Server\YOUR-MSSQL-SERVER-INSTANCE-FOLDER\PYTHON_SERVICES"
+	python.exe -c "import revoscalepy"
+
+You have to add this folder (`C:\Program Files\Microsoft SQL Server\YOUR-MSSQL-SERVER-INSTANCE-FOLDER\PYTHON_SERVICES`) to the windows path. 
+
+Make sure that python runs inside SQL Server correctly. For that, open SQL Server and connect to your database engine. You need to enable external scripts. For that, press `New Query` and execute: 
+
+	Exec sp_configure 'external scripts enabled', 1
+	Reconfigure with override
+
+Then you need to restart SQL Server. Right click on the server and then restart. After restarting, you have to make sure that in external scripts enables run_value is 1.
+
+Finally, to test that everything works correctly you have to execute the file [sqlpytest.sql](sql/sqlpytest.sql).
+
+You need to change the user and password in SQL Server. For that you have to execute the file `change_auth.sql` (changing the variables user and password).
+
+#### Install the rest of the libraries needed to run the the demo
+The next step is to install several libraries that we need to run the demo. First you need to install OpenCV:
+
+	cd "C:\Program Files\Microsoft SQL Server\YOUR-MSSQL-SERVER-INSTANCE-FOLDER\PYTHON_SERVICES\Scripts"
+	conda.exe install -c https://conda.binstar.org/conda-forge opencv -y
```

Comment:
I actually ran into the following error when running this line, and found a solution by setting the environment variable CONDA_SSL_VERIFY=false from this link (https://github.com/conda/conda/issues/5170):
Fetching package metadata ...ERROR urllib3.connection:_match_hostname(360): Cert
ificate did not match expected hostname: conda.binstar.org. Certificate: {'subje
ct': ((('commonName', 'anaconda.com'),),), 'subjectAltName': [('DNS', 'anaconda.
com'), ('DNS', 'anacondacloud.com'), ('DNS', 'anacondacloud.org'), ('DNS', 'bins
tar.org'), ('DNS', 'wakari.io')]}

Reply:
Fixed